src/libs/actions/Welcome.js

```diff
@@ -36,7 +37,11 @@ Onyx.connect({
     key: ONYXKEYS.NVP_IS_FIRST_TIME_NEW_EXPENSIFY_USER,
     initWithStoredValues: false,
     callback: (val) => {
-        isFirstTimeNewExpensifyUser = val;
+        // If this is a first time new expensify user, let's only update the state of isFirstTimeNewExpensifyUser from true to false
```

Comment:
```suggestion
        // If this is a first time new Expensify user, let's only update isFirstTimeNewExpensifyUser from true to false
```
having a mention of "state" is tripping me up a little bit since it's not a state variable (in a react sense). Just my opinion to make it simpler.